### PR TITLE
[16.0] [ADD] [FIX] fix map render with change timeout value

### PR DESCRIPTION
**Pull request by igr-ok**

### CODE REVIEW

**Comment by legalsylvain** [comment]
Hi. could you squash commits : 
- git rebase -i HEAD~3
- s option on two last commits.
thanks !

**Review by Jordi-Buitrago — APPROVED** [review]

**Review by navarrorico — APPROVED** [review]

File: web_view_leaflet_map/i18n/fr.po [hunk 1]
```diff
@@ -119,3 +54,30 @@ msgstr "Vue"
 #: model:ir.model.fields,field_description:web_view_leaflet_map.field_ir_ui_view__type
 msgid "View Type"
 msgstr "Type de Vue"
+
+#~ msgid "Activity"
+#~ msgstr "Activité"
+
+#~ msgid "Calendar"
+#~ msgstr "Calendrier"
```
Comment (legalsylvain) [inline web_view_leaflet_map/i18n/fr.po]: All that addition looks weird. Why add commented translation ? 
could you revert changes in fr.po ? 
thanks !

Comment (igr-ok) [reply web_view_leaflet_map/i18n/fr.po]: Yes, changes only in map_renderer.js

**Comment by navarrorico** [comment]
@legalsylvain Can we reopen this? the issue still there and is necessary to this module for work properly. 
 
 What else need to being merged?
 
 Thank you!

**Comment by legalsylvain** [comment]
Hi. 
- only @igr-ok or @OCA/geospatial-maintainers can reopen the PR.

regards.

**Pull request closed without merging**